lib/zendesk_apps_tools/server.rb

```diff
@@ -44,6 +43,18 @@ class Server < Sinatra::Base
       ZendeskAppsSupport::Installed.new([app_js], [installation]).compile_js
     end
 
+    get "/:file" do |file|
+      access_control_allow_origin
+      send_file File.join(settings.root, 'assets', file)
```

Comment:
it would probably be best if this didn't support `assets/subdirectory/../file_in_root_dir.png` because that won't work when you upload the app